packages/panzer/mini-em/example/BlockPrec/main.cpp

```diff
@@ -712,8 +721,28 @@ int main_(Teuchos::CommandLineProcessor &clp, int argc,char * argv[])
     auto xmlOut = stacked_timer->reportWatchrXML(test_name + ' ' + std::to_string(comm->getSize()) + " ranks", comm);
     if(xmlOut.length())
       std::cout << "\nAlso created Watchr performance report " << xmlOut << '\n';
-  } else
+
+    if ( print_fom && (comm->getRank() == 0) ) {
+      std::string fom_timer_name = "Mini-EM@Mini-EM: Total Time@Mini-EM: timestepper@Mini-EM: Advance Time Step@Stratimikos: BelosLOWS";
```

Comment:
Can we add a check for this 
      <Parameter name="Mesh Type" type="string" value="tet"/>
to make sure we don't use the wrong maxwell.xml which is a hex mesh?

Reply:
Ok. That's great